packages/experiments/src/components/Button/Actionable/Actionable.types.ts

```diff
@@ -107,7 +108,7 @@ export interface IActionableProps
 /**
  * {@docCategory Actionable}
  */
-export interface IActionableViewProps extends IActionableProps {
+export interface IActionableViewProps extends ISlottableProps<IActionableSlots>, Omit<IActionableProps, keyof IActionableSlots> {
```

Comment:
Hmm, this seems pretty smelly. I'd like to understand more of what's going on here.. I don't think this is a good thing to require users to do as-is.